app/services/CheckoutService.scala

```diff
@@ -22,10 +22,14 @@ class CheckoutService(identityService: IdentityService, salesforceService: Sales
       identityService.registerGuest(subscriptionData.personalData)
     }
 
+    //TODO when implementing test-users this requires updating to supply data to correct location
+    val touchpointBackend = TouchpointBackend.Normal
+    val ratePlan = touchpointBackend.ratePlan.ratePlanId
+
     for {
       idUser <- idUserF
       memberId <- salesforceService.createOrUpdateUser(subscriptionData.personalData, idUser)
```

Comment:
Turns out we don't need to do this as the service calls TouchpointBackend: https://github.com/guardian/subscriptions-frontend/blob/master/app/services/SalesforceService.scala#L34-36